README.md

```diff
@@ -17,7 +17,6 @@ Name | Description
 * [SOME/IP Provider](https://github.com/eclipse-kuksa/kuksa-someip-provider)
 * [DDS Provider](https://github.com/eclipse-kuksa/kuksa-dds-provider)
 * [CSV Provider](https://github.com/eclipse-kuksa/kuksa-csv-provider)
-* [eCAL Provider](https://github.com/eclipse-kuksa/kuksa-incubation/tree/main/ecal2val)
```

Comment:
I assume this is a merge conflict error, i.e. that it is not intentional to remove this line?